lua/neo-tree/defaults.lua

```diff
@@ -200,6 +200,8 @@ local config = {
       folder_open = "",
       folder_empty = "󰉖",
       folder_empty_open = "󰷏",
+      folder_symlink = "",
+      file_symlink = "",
```

Comment:
This link is almost impossible for me to distinguish from the standard "unknown file" icon. Can we find one that's more obvious?

Reply:
Also, I'm not entirely sure I even want a default icon set for this. Some people might want to see the target's icon and not a generic symlink icon.